integration/containerd/confidential/lib.sh

```diff
@@ -309,3 +344,14 @@ configure_cc_containerd() {
 	iptables -P FORWARD ACCEPT
 }
 
+#
+# Internal functions.
+#
+
+# Export the RUNTIME_CONFIG_PATH variable if it not set already.
+#
+_load_RUNTIME_CONFIG_PATH() {
```

Comment:
Could we expanded the use of this to a few other methods to reduce duplicated code? eg in `enable_agent_debug`, `enable_agent_console`, `enable_runtime_debug`, `enable_full_debug` & `switch_image_service_offload`. I've mentioned creating a new internal function for this to my team in their `ccv0.sh` re-factor, but you've already done it, which is awesome!

Reply:
Yes, I was thinking in do that refactor. I am also going to use the new `add_kernel_params` in the `switch_image_service_offload` and any other methods which change kernel parameters. I will send it in the v2.

Reply:
That looks good. Thank you!